humble/humble_here

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+IP="$1"
+
+if [ -z "$1" ]
+    then
+        echo "No argument given, If an IP is required, please provide it as an argument. Using [localhost] for this run."
+        IP="localhost"
+fi
+
+xhost +local:root 
+
+XAUTH=/tmp/.docker.xauth
+
+if [ ! -f $XAUTH ]
+then
+    xauth_list=$(xauth nlist :0 | sed -e 's/^..../ffff/')
+    if [ ! -z "$xauth_list" ]
+    then
+        echo $xauth_list | xauth -f $XAUTH nmerge -
+    else
+        touch $XAUTH
+    fi
+    chmod a+r $XAUTH
+fi
+
+docker run -it \
+  --rm \
+  --privileged \
+  --net=host \
+  --env="DISPLAY=$DISPLAY" \
+  --env="QT_X11_NO_MITSHM=1" \
+  --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" \
+  --env="XAUTHORITY=$XAUTH" \
+  --env="ROS_MASTER_URI=http://$IP:11311" \
```

Comment:
I don't believe ROS2 uses master_uri or hostname. 

Please adjust this file so that you can pass in the ROS domain ID
https://docs.ros.org/en/humble/Concepts/About-Domain-ID.html